packages/SwingSet/test/test-transcriptlessness.js

```diff
@@ -0,0 +1,57 @@
+/* global require */
+// eslint-disable-next-line import/order
+import { test } from '../tools/prepare-test-env-ava';
+
+import { provideHostStorage } from '../src/hostStorage';
+import { buildVatController } from '../src/index';
+
+function capdata(body, slots = []) {
+  return harden({ body, slots });
+}
+
+function capargs(args, slots = []) {
+  return capdata(JSON.stringify(args), slots);
+}
+
+async function testTranscriptlessness(t, useTranscript) {
+  const config = {
+    bootstrap: 'bootstrap',
+    vats: {
+      bootstrap: {
+        sourceSpec: require.resolve('./vat-transcript-maybe.js'),
+        creationOptions: {
+          enableSetup: true,
+          useTranscript,
+        },
+      },
+    },
+  };
+  const hostStorage = provideHostStorage();
+  const c1 = await buildVatController(config, [], {
+    hostStorage,
+  });
+  await c1.run();
+  t.deepEqual(c1.dump().log, ['ephemeralCounter=1 sturdyCounter=1']);
+
+  const c2 = await buildVatController(config, [], {
+    hostStorage,
+  });
+  c2.queueToVatExport('bootstrap', 'o+0', 'go', capargs([]), 'panic');
+  await c2.run();
+  if (useTranscript) {
+    t.deepEqual(c2.dump().log, [
+      'ephemeralCounter=1 sturdyCounter=1',
+      'ephemeralCounter=2 sturdyCounter=2',
+    ]);
+  } else {
+    t.deepEqual(c2.dump().log, ['ephemeralCounter=1 sturdyCounter=2']);
+  }
+}
+
+test('transcript on', async t => {
+  await testTranscriptlessness(t, true);
+});
+
+test('transcript false', async t => {
```

Comment:
nit: parallelism between the two test names. Maybe `transcript on` and `transcript off`?

Reply:
Yes, of course it's `transcript off`.  Look over there!  It's Hailey's Comet!